SocialCareCaseViewerApi/V1/Boundary/Response/ListRelationshipsV1Response.cs

```diff
@@ -0,0 +1,11 @@
+using SocialCareCaseViewerApi.V1.Domain;
+
+namespace SocialCareCaseViewerApi.V1.Boundary.Response
+{
+    public class ListRelationshipsV1Response
+    {
+        public long PersonId { get; set; }
+
+        public PersonalRelationshipsV1<RelatedPerson> PersonalRelationships { get; set; } = new PersonalRelationshipsV1<RelatedPerson>();
```

Comment:
Would it be worth renaming the `RelatedPerson` domain object as well to have complete separation between old and new objects.

Reply:
Ahhhh yeah, I think I thought that wouldn't change or something but I think it will do now.

Reply:
Done!